catalyst/contrib/datasets/mnist.py

```diff
@@ -1,5 +1,6 @@
 # flake8: noqa
```

Comment:
could we remove noqa?

Reply:
ok

Reply:
no, it is still needed for copypasted mnist